src/routers/AppRouter.tsx

```diff
@@ -18,102 +18,145 @@ import VedtakContainer from "../components/vedtak/container/VedtakContainer";
 import DialogmoteInnkallingContainer from "../components/dialogmote/innkalling/DialogmoteInnkallingContainer";
 import { erLokalEllerPreprod } from "../utils/miljoUtil";
 import AvlysDialogmoteContainer from "../components/dialogmote/avlys/AvlysDialogmoteContainer";
+import { useValgtPersonident } from "../hooks/useValgtBruker";
+import { useDispatch } from "react-redux";
+import {
+  hentAktivBruker,
+  pushModiaContext,
+} from "../data/modiacontext/modiacontext_actions";
+import AppSpinner from "../components/AppSpinner";
+import { useAppSelector } from "../hooks/hooks";
 
-const AppRouter = () => {
-  const fnr = window.location.pathname.split("/")[2];
-  if (!erGyldigFodselsnummer(fnr)) {
-    return (
-      <Router>
-        <Route path="*" component={IngenBrukerContainer} />
-      </Router>
-    );
-  }
+const useFnrFromParams = (): string => {
+  return window.location.pathname.split("/")[2];
+};
+
+const setAktivBrukerFromParams = (brukerIdent: string) => {
+  const dispatch = useDispatch();
+  console.log("setAktivBrukerFromParams");
+  dispatch(
+    pushModiaContext({
+      verdi: brukerIdent,
```

Comment:
Kan evnt vurdere å endre input i pushModiaContext action fra data: any til interface og enum, for å sikre riktige verdier

Reply:
Ja, hadde vært fint med type på data i ModiaContextState og actions når man først er inne og endrer her :)

Reply:
Jepp, skal legge det til 👍

Reply:
Lagt til interface.